js/compiler/igor-controller.js

```diff
@@ -72,20 +79,43 @@ function job_remove(job) {
  * Select the flags for Igor to run the job.
  * @param {GMLProject} project
  * @param {string} runtime_path
+ * @param {string?} user_path
  * @param {IgorSettings} settings
  * @returns {Result<string[]>}
  */
-function job_flags_get(project, runtime_path, settings) {
+function job_flags_get(project, runtime_path, user_path, settings) {
+
+    /**
+     * @type {{[K in IgorPlatform]?: string}}
+    */
+    const exts = {
+        Windows: ".zip",
+        Mac: ".zip",
+        Linux: ".appimage",
+    };
+
+    let projectName = $gmedit['gml.Project'].current.name;
```

Comment:
I'd prefer this uses `project_current_get()` from utils rather than querying directly - genuinely just a nitpick that i think looks nicer and clearer.